test/factories/teacher_set_factory.rb

```diff
@@ -0,0 +1,5 @@
+class Cranky::Factory
```

Comment:
This is interesting...  Does a teacher set only need a title?

Reply:
And any books need to be valid.  It has this line:
`validates_associated :books`